docs/upgrade/16_to_17/add_is_published.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/sh
+
+# Elasticsearch details
+ES_HOST="http://localhost:9200"
+INDEX_NAME="opencast_event"
+FIELD_NAME="is_published"
+
+# Elasticsearch authentication
+ES_USER="your_username"
+ES_PASS="your_password"
+
+ES_HEALTH=$(curl -s -u "$ES_USER:$ES_PASS" -X POST "$ES_HOST/_cluster/health" | jq -e '.status')
```

Comment:
`jq` is not installed by default. Maybe add a check at the beginning if `jq` is available and fail fast with a message to install this dependency?